SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml

```diff
@@ -119,7 +119,8 @@ stages:
 
       - set:
           event.type: [ 'connection', 'access', 'allowed' ]
-        filter: '{{set_ecs_fields.event.type == ["allowed"] or parse_kv.message.result == "OBSERVED"}}'
+        filter: '{{ parse_kv.message.block_id == "0" or set_ecs_fields.event.type == ["allowed"] or parse_kv.message.result == "OBSERVED"}}'
```

Comment:
No test associated to this fix?

Reply:
https://github.com/SEKOIA-IO/intake-formats/blob/main/SkyhighSecurity/skyhigh_secure_web_gateway/tests/mcafee_access_log_success.json#L19-L23

This is a fix for this part of the test